server.js

```diff
@@ -0,0 +1,204 @@
+var config = require('./config')
+var log = require('./log')
+var subscriptions = require('./subscriptions')
+var notifier = require('./notifier')
+var hapi = require('hapi')
+var joi = require('joi')
+
+var db = require('./db/' + config.db.driver)
+
+var server = new hapi.Server()
+server.connection({
+  host: config.server.host,
+  port: config.server.port
+})
+
+server.route([
+  {
+    method: 'POST',
+    path: '/v1/publish',
```

Comment:
I suggest we explicitly make this a `v0` API, based on stategy discussions yesterday